protocol-units/da-sequencer/node/src/main.rs

```diff
@@ -27,12 +32,16 @@ async fn main() -> Result<(), Box<dyn Error>> {
 		Godfig::new(ConfigFile::new(config_file), vec![]);
 	let da_sequencer_config: DaSequencerConfig = godfig.try_wait_for_ready().await?;
 
+	// Initialize whitelist
+	let mut whitelist_path = dot_movement.get_path().to_path_buf();
+	whitelist_path.push("default_signer_address_whitelist");
```

Comment:
Put the path in the config, the same in relative to dot_movement. Use the `da-sequence`r config base path to have all `da-sequencer` file in the same directory.